src/Maker/MakeEntity.php

```diff
@@ -106,6 +111,14 @@ public function interact(InputInterface $input, ConsoleStyle $io, Command $comma
         $value = $io->askQuestion($question);
 
         $input->setArgument('name', $value);
+
+        if (class_exists(ApiResource::class) && !$input->getOption('api-resource')) {
+            $description = $command->getDefinition()->getOption('api-resource')->getDescription();
+            $question = new ConfirmationQuestion($description, false);
+            $value = $io->askQuestion($question);
+
+            $input->setOption('api-resource', $value);
+        }
```

Comment:
We should only ask this question when the entity does NOT exist yet. Unfortunately, that means we'll need to duplicate a bit of logic from `generate()` up here:

```php
$entityClassDetails = $generator->createClassNameDetails(
    $input->getArgument('name'),
    'Entity\\'
);
$classExists = class_exists($entityClassDetails->getFullName());
```

Reply:
Fixed in d003a487f5f47f99b3276ed45c21f3958d27dffa (with some more Voodoo...)